configure.ac

```diff
@@ -199,6 +199,7 @@ AC_CHECK_HEADERS([ \
 	X11/X.h \
 	X11/Xlib.h \
 	zlib.h \
+	stdbool.h \
```

Comment:
Please maintain the alphabetical order of this list and move this to before stddef.h